docs/docs/widgets/divider.md

```diff
@@ -50,13 +50,11 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 ## Styles
 
-| <div style={{ width:"100px"}}> Style </div> | <div style={{ width:"100px"}}> Description </div>                                               |
-| ------------------------------------------- | ----------------------------------------------------------------------------------------------- |
-| Divider Color                               | It is used to set the color of the divider. Use hex code to set the background color.           |
-| Visibility                                  | This property is used to set the visibility of the divider. The property accepts Boolean value. |
-
-:::info
-Any property having **fx** button next to its field can be **programmatically configured**.
-:::
+| <div style="width:100px;"> Property </div>    | <div style="width:100px;"> Description </div> | <div style="width:100px;"> Configuration Options </div> |
+| ------------------------------------------- | ----------------------------------------------- | ------------------------------------------------------ |
+| Divider Color | Sets the background color of the component. | Select the color or click on fx and input code that programmatically returns a HEX color code. |
+| Visibility  | Controls component visibility.Toggle or set dynamically  | Enable/disable the toggle button or dynamically configure the value by clicking on fx and entering a logical expression.
```

Comment:
Here you need to add a full stop at the end of the sentence in description, also give a space before `Toggle`.